qiita_db/reference.py

```diff
@@ -23,6 +79,7 @@ class Reference(QiitaObject):
     sequence_fp
     taxonomy_fp
     tree_fp
+    sortmerna_indexed_db
```

Comment:
I don't know if we really want to have different reference classes for 16S, 18S, proteins, etc...
I think I just need to change the property to return None if that is not used. The taxonomy_fp and the tree_fp are also optional (e.g. ITS does not have a tree), so they should also return None if that doesn't exist. That is a bug on the code and I'm going to fix it (right now it fails if they're not present). I'll also add tests for them.